long-term-storage-aws.md

```diff
@@ -1,13 +1,24 @@
-AWS Long Term Storage
-=====================
+AWS Multi-Cluster Storage Configuration
```

Comment:
Technically speaking, someone may want to use durable storage and not multi-cluster.

Reply:
In other words I think we should keep the same title.